docs/decisions/adr-000-document-architecture-decisions.md

```diff
@@ -0,0 +1,33 @@
+# ADR 000: Document architecture decisions
+
+## Status
+
+Proposed
+
+## Context
+
+It can be difficult to keep track of why software has been architected in a particular way.
+
+We aim to document as many decisions as possible through [our RFC process](https://docs.google.com/document/d/1_Vm7nFfqZMNx5gEJ1lTzlwi79WnrS6pXkoOvFG4wFxM/edit), but it's difficult if not impossible to anticipate all the decisions that will need to be made through the entire course of a project. This leaves many of our projects with documentation that may have been up-to-date and thorough at the time an RFC was written, but quickly becomes incomplete or inaccurate as implementation progresses.
```

Comment:
WDYT about changing this to the RFC template page (for more context) rather than the example rfc?
```suggestion
We aim to document as many decisions as possible through [our RFC process](https://works.hashicorp.com/articles/rfc-template), but it's difficult, if not impossible, to anticipate all the decisions that will need to be made through the entire course of a project. This leaves many of our projects with documentation that may have been up-to-date and thorough at the time an RFC was written, but quickly becomes incomplete or inaccurate as implementation progresses.
```

Reply:
maybe parens instead of commas?

```diff
- [our RFC process](https://works.hashicorp.com/articles/rfc-template), but it's difficult, if not impossible,
+ [our RFC process](https://works.hashicorp.com/articles/rfc-template), but it's difficult(if not impossible)
```

Reply:
Great suggestions! I updated the link to the `works.hashicorp.com` page, and ended up removing the `if not impossible` 👍